freeipa/src/main/resources/schema/app/20190516151400_add_tenant_to_stack.sql

```diff
@@ -0,0 +1,32 @@
+-- // Create Changelog
+
+-- Default DDL for changelog table that will keep
+-- a record of the migrations that have been run.
+
+-- You can modify this to suit your database before
+-- running your first migration.
+
+-- Be sure that ID and DESCRIPTION fields exist in
+-- BigInteger and String compatible fields respectively.
+
+ALTER TABLE stack
+    ADD tenant VARCHAR(255);
+
+CREATE UNIQUE INDEX tenant_uindex
+    ON stack (tenant);
+
+CREATE UNIQUE INDEX environment_tenant_uindex
+    ON stack (name, environment, tenant);
+
+CREATE UNIQUE INDEX stack_name_environment_tenant_uindex
```

Comment:
There was probably a unique index on name and environment. should we drop uniqueness there that in favor of the name_environment_tenant unique index?

Reply:
Removed this index, not required. .

Reply:
I meant to remove the other unique index on name and environment and keep this unique index on name, environment, and tenant.